File: Excel/498 Challenge.R
library(tidyverse)
library(readxl)

path = "Excel/498 Soccer Champions Alignment.xlsx"
input = read_xlsx(path, range = "A2:B10")
test = read_xlsx(path, range = "D2:E24")

result = input %>%
  separate_rows(Years, sep = ",") %>%
  mutate(Years = as.numeric(Years)) %>%
  arrange(desc(Years)) %>%
  select(Year = Years, Winner = Winners)

identical(result, test)
# [1] TRUE

File: Excel/400-499/498/498 Challenge.py
import pandas as pd

path = "498 Soccer Champions Alignment.xlsx"
input = pd.read_excel(path, usecols= "A:B", skiprows= 1, nrows = 8)
test = pd.read_excel(path, usecols= "D:E", skiprows= 1)

result = input.copy()
result['Years'] = result['Years'].str.split(', ')
result = result.explode('Years').astype({'Years': 'int64'})\
    .sort_values('Years', ascending=False).reset_index(drop= True)\
    .rename(columns = {'Years': 'Year', 'Winners': 'Winner'})\
    .reindex(columns = ['Year', 'Winner'])

print(result.equals(test)) # True